src/collectors/mod.rs

```diff
@@ -30,25 +31,25 @@ impl CollectorScheduler {
 
     pub fn start(&self, sender: Sender<Message>) {
         if self.collectors.is_empty() {
-            println!("No collectors configured!");
+            error!("No collectors configured!");
```

Comment:
It would be good to log this in "critical" level since it causes the application to exit. I noticed that there is no macro for that. Did you check if it is possible to define our own levels with `log`? [slog](https://github.com/slog-rs/slog) seems to support the standard syslog levels. Did you consider using that?

Reply:
As discussed, lets go with `log` crate for now as it seems to be the "standard" way.